python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -536,3 +563,150 @@ def to_arrow(self):
                 arrow_polygons,
             ],
         )
+
+    def contains(self, other, align=True):
+        """Compute from a set of points and a set of polygons which points fall
+        within each polygon. Note that `polygons_(x,y)` must be specified as
+        closed polygons: the first and last coordinate of each polygon must be
+        the same.
+
+        This implements `.contains_properly`, which shares a large
+        space of correct cases with `GeoPandas.contains` but they do not
+        produce identical results. In the future we will use intersection
+        testing to match .contains behavior.
+
+        Parameters
+        ----------
+        test_points_x
+            x-coordinate of test points
+        test_points_y
+            y-coordinate of test points
```

Comment:
Just to be more clear which points these are.
```suggestion
            x-coordinates of points to test for containment
        test_points_y
            y-coordinates of points to test for containment
```

Reply:
Done.